docs/streams/index.html

```diff
@@ -18,10 +18,24 @@
 <script><!--#include virtual="../js/templateData.js" --></script>
 
 <script id="streams-template" type="text/x-handlebars-template">
-    <h1>Kafka Streams API</h1>
+    <h1>Kafka Streams</h1>
 
     <h3 style="max-width: 75rem;">The easiest way to write mission-critical real-time applications and microservices with all the benefits of Kafka's server-side cluster technology.</h3>
 
+    <p>The Streams API of Apache Kafka, available through a Java library, can be used to build highly scalable, elastic, fault-tolerant, distributed applications and microservices. With the Kafka Streams API you can create real-time applications that power your core business. It is the easiest yet the most powerful technology to process data stored in Kafka. It builds upon important concepts for stream processing such as efficient management of application state, fast and efficient aggregations and joins, properly distinguishing between event-time and processing-time, and seamless handling of late-arriving and out-of-order data.</p>
+
+    <p>A unique feature of the Kafka Streams API is that the applications you build with it are normal Java applications. These applications can be packaged, deployed, and monitored like any other Java application – there is no need to install separate processing clusters or similar special-purpose and expensive infrastructure!</p>
+
+    <h2>Documentation</h2>
+
+    <ul>
+    <li><a href="/documentation/streams/quickstart">Quickstart</a></li>
```

Comment:
Note that we have two quickstarts right now:
- Write Your First App: https://kafka.apache.org/0110/documentation/streams/tutorial
- Playw With Demo App: https://kafka.apache.org/0110/documentation/streams/quickstart

We're also revamping the Streams landing page right now.  I'd suggest to sync with @derrickdoo on the changes in this PR of yours.

Reply:
I talked with @derrickdoo  about this yesterday - neither of those files match up to those titles:
- `Play with a Streams Application` = https://kafka.apache.org/0110/documentation/streams/quickstart
- `Write your own Streams Applications` = https://kafka.apache.org/0110/documentation/streams/tutorial

We need to be consistent when referring to these topics. So we should either rename these topics to match the titles as you've suggested (`Write Your First App` and `Play With Demo App`); or rename the references to match the current topic titles.

Reply:
Also, `/documentation/streams/quickstart` is linked again a [few lines below](https://github.com/apache/kafka/pull/3862/files/495e64f4740bdc9c9231ca88f672496b6d3ecfb3#diff-ee4356d82f838700797583e3ba114b2fR43).

Reply:
Agreed, we need consistent naming.  But I would suggest to NOT change the URLs however because they are already used and referred to in the wild.

Reply:
@miguno yeah, i agree about the URLs - that opens a pandora's box of redirect responsibilities.

Reply:
@joel-hamill @miguno I'm fine merging this stuff in for now, but lets just make sure to double check the new design for this page and see if any of this needs to be incorporated on there.

Reply:
Original plan is that moving forward we can merge these two in a single series under one section: i.e. first you write the app, and then execute it and see the results, and then (in the future plan) adding a few more code (like IQ, etc) and execute again..